include/irr/asset/CTexturePacker.h

```diff
@@ -317,13 +373,23 @@ class ICPUTexturePacker : protected ITexturePacker
     ICPUImage* getImage() { return m_physAddrTex.get(); }
 
 private:
+    uint32_t countLevelsTakingAtLeastOnePage(const VkExtent3D& _extent, const ICPUImage::SSubresourceRange& _subres)
+    {
+        uint32_t levelCount = 1u;
+        for (uint32_t i = 0u; (_extent.width>>(_subres.baseMipLevel+i) >= m_pgSzxy) || (_extent.height>>(_subres.baseMipLevel+i) >= m_pgSzxy); ++i)
+            levelCount = i+1u;
+        return std::min(_subres.levelCount, levelCount);
```

Comment:
I though there as a simple equation for getting this without having to resort to a trial&error loop